src/tests/daos_perf.c

```diff
@@ -88,13 +89,94 @@ bool			 ts_nest_iterator;
 bool			ts_rebuild_only_iteration = false;
 /* rebuild without update */
 bool			ts_rebuild_no_update = false;
+/* test inside ULT */
+bool			ts_in_ult = false;
+bool			ts_profile_vos = false;
```

Comment:
(style)  do not initialise globals to false

Reply:
any reason not to remove the explicit false?...Not sure I care much